tests/test_pacing.py

```diff
@@ -18,6 +20,15 @@ def use(self):
         pass
 
 
+class NonesyQuestion(Answerable):
```

Comment:
Questions don't inherit from `Answerable`, so i'd prefer if we remove this so we can make it more real-like.

Reply:
They don't in the primary library, but should they be prevented from doing so?

Reply:
fixed